templates/zerver/api/server-settings.md

```diff
@@ -0,0 +1,70 @@
+# Get global settings
+
+Fetch global settings for the Zulip server.
+
+`GET {{ api_url }}/v1/server_settings`
+
+## Usage examples
+
+<div class="code-section" markdown="1">
+<ul class="nav">
+<li data-language="python">Python</li>
+<li data-language="curl">curl</li>
+</ul>
+<div class="blocks">
+
+<div data-language="curl" markdown="1">
+
+```
+curl {{ api_url }}/v1/server_settings \
+    -u BOT_EMAIL_ADDRESS:BOT_API_KEY
+```
+
+</div>
+
+<div data-language="python" markdown="1">
+
+{generate_code_example(python)|/server_settings:get|example}
+
+</div>
+
+</div>
+
+</div>
+
+## Arguments
+
+{generate_api_arguments_table|zulip.yaml|/server_settings:get}
+
+## Response
+
+#### Return values
+
+* `authentication_methods`: object in which each key-value pair in the object
+  indicates if the authentication method in the key is enabled or not.
+* `zulip_version`: the version of Zulip running in the server.
+* `push_notifications_enabled`: whether mobile/push notifications are enabled
+  or not.
+* `email_auth_enabled`: setting for allowing users authenticate with an
+  email-password combination.
+* `require_email_format_usernames`: whether usernames should have an email
+  address format or not.
+* `realm_uri`: the URI where the organization can be found.
+* `realm_name`: the organization's name.
+* `realm_icon`: URI where the organization's icon can be found.
+* `realm_description`: the description for the realm, as shown in the login
+  page.
+
+Please note that `email_auth_enabled`, `require_email_format_usernames`,
+`realm_uri`, `realm_name`, `realm_icon` and `realm_description` might not be
+present in the response.
+
+If they are present, that means that the server has only one organization; but if
+there are not, that means that there are more than one organization and therefore the
+values might differ depending on the organization.
```

Comment:
We should explain why: "Please note that not all of these attributes are guaranteed to appear in a response, for two reasons.  First, this endpoint has evolved over time, so responses from older Zulip servers might be missing some keys (in which case a client should assume the appropriate default)", and second, if a `/server_settings` request is made to the root domain of a multi-subdomain server like the root domain of zulipchat.com, the settings that are realm-specific are not known and thus not provided.